### DISCUSSION TITLE
付费后仍然显示是

### DISCUSSION THREAD

**Discussion by lnd1220** [getlantern/lantern-forum-cn#568] [General]
付费后依然显示是免费版，用了很多年了
账号是lnd@vip.sina.com
![86c09431867c5d1ec998f8005fdfb78](https://github.com/getlantern/lantern-forum-cn/assets/45390777/8c740fb6-3747-4879-bf9e-857c2746e8a2)
![86c09431867c5d1ec998f8005fdfb78](https://github.com/getlantern/lantern-forum-cn/assets/45390777/8c740fb6-3747-4879-bf9e-857c2746e8a2)
![86c09431867c5d1ec998f8005fdfb78](https://github.com/getlantern/lantern-forum-cn/assets/45390777/8c740fb6-3747-4879-bf9e-857c2746e8a2)

**Comment by lnd1220** [comment]
![Uploading 微信图片_20240613224735.jpg…]()

**Comment by MaxMKMax** [comment]
@lnd1220 您好. 请放心, 我帮您安排. 下周给您答复. 谢谢您的耐心等待.
